### ISSUE TITLE
make it more obvious which players have been eliminated

### ISSUE THREAD

**Issue by tsj845** [radicise/territopple#55]
this should be done by changing something about the joined list (eg. dimming the names of eliminated players)

**Comment by david-aashish** [comment]
Can I take this issue?
